api/yelp.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+export default axios.create({
```

Comment:
question: are we moving away from yelp?
we might be able to move the data into db_static.json.